.github/workflows/ci.yaml

```diff
@@ -54,130 +49,93 @@ jobs:
         command: check ${{ matrix.checks }}
 
   sort:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-22.04
     needs:
       - "rustfmt"
       - "markdown-lint"
     steps:
       - uses: actions/checkout@v3
-      - uses: actions-rs/toolchain@v1
+      - uses: dtolnay/rust-toolchain@master
         with:
-          toolchain: stable
-          profile: minimal
-      - uses: actions-rs/install@v0.1
-        with:
-          crate: cargo-sort
-          version: latest
-          use-tool-cache: true
-      # We run this manually because actions-rs/cargo doesn't have output redirect
+          toolchain: nightly-2022-07-22
```

Comment:
We probably want the `stable` toolchain for this.

Reply:
removed the toolchain action from both. By default the images more or less track stable.